docs/howto_guides/api_gui.md

```diff
@@ -0,0 +1,161 @@
+<br/>
+
+Each endpoint is unique and requires the applicable attribute names to be known in order to structure a query in a meaningful way.<br/>
+<br/>
+
+![find get studies](../_static/images/howto_guides/api_gui/find_get_studies.png)
+The `GET /studies` endpoint is a general purpose way to retrieve NMDC studies based on parameters provided by the user. Studies can be filtered and sorted based on the applicable [Study attributes](https://microbiomedata.github.io/nmdc-schema/Study/). <br/>
+<br/>
+
+![find get studies by study_id](../_static/images/howto_guides/api_gui/find_get_studies_study_id.png)
+If the study identifier is known, a study can be retrieved directly using the `GET /studies/{study_id}` endpoint. Note that only one study can be retrieved at a time using this method.<br/>
+<br/> 
+
+![find get biosamples](../_static/images/howto_guides/api_gui/find_get_biosamples.png)
+The `GET /biosamples` endpoint is a general purpose way to retrieve biosample metadata using user-provided filter and sort criteria. Please see the applicable [Biosample attributes](https://microbiomedata.github.io/nmdc-schema/Biosample/).<br/>
+<br/>
+
+![find get biosamples by sample_id](../_static/images/howto_guides/api_gui/find_get_biosamples_sample_id.png)
+If the biosample identifier is known, a biosample can be retrieved directly using the `GET /biosamples/{sample_id}`. Note that only one biosample metadata record can be retrieved at a time using this method.<br/>
+<br/>
+
+![find get data objects](../_static/images/howto_guides/api_gui/find_get_data_objects.png)
+To retrieve metadata about NMDC data objects (such as files, records, or omics data) the `GET /data_objects` endpoint may be used along with various parameters. Please see the applicable [Data Object attributes](https://microbiomedata.github.io/nmdc-schema/DataObject). <br/>
+<br/>
+
+![find get data objects by data object_id](../_static/images/howto_guides/api_gui/find_data_objects_data_object_id.png)
+If the data object identifier is known, the metadata can be retrieved using the `GET /data_objects/{data_object_id}` endpoint. Note that only one data object metadata record may be retrieved at a time using this method.<br/>
+<br/>
+
+![find get activities](../_static/images/howto_guides/api_gui/find_get_activities.png)
+The `GET /activities` endpoint is a general way to fetch metadata about various activities (e.g. metagenome assembly, natural organic matter analysis, library preparation, etc.). Any "slot" (a.k.a. attribute) for [WorkflowExecutionActivty](https://microbiomedata.github.io/nmdc-schema/WorkflowExecutionActivity/) or [PlannedProcess](https://microbiomedata.github.io/nmdc-schema/PlannedProcess/)  may be used in the filter and sort parameters, including attributes for subclasses of `WorkflowExecutionActvity` and `PlannedProcess`, such as slots used in the `MetabolomicsAnalysisActivity` or `Extraction` class among others.<br/>
```

Comment:
Regarding the final sentence in this paragraph:

> Any "slot"...

I had a hard time understanding that particular sentence. I wonder whether it could be split into two.

Reply:
I fixed this. I think there was a similar comment above. Let me know if it is still difficult to understand.